website/docs/providers/notifier_provider.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: NotifierProvider
+---
+
+import Tabs from "@theme/Tabs";
```

Comment:
This page probably should include AsyncNotifier too. They are the same, with AsyncNotifier being the most used one

Reply:
Updated the tittle as `(Async)NotifierProvider`